discord/installer/install_linux.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+INSTALL_OPTIONS=("Download correct node and install" "Install using local version" "Exit")
+
+runInstaller() {
+    echo "Please enter the install directory";
+    read INSTALL_DIR;
+    echo "Installing to $INSTALL_DIR";
+
+    $DIR/node $DIR/index.js $DIR $INSTALL_DIR;
+}
+
+#TODO:  Confirm this is cool with the user
+downloadNodeAndInstall() {
+    echo "Downloading standalone tarball..." 
+    curl -o $DIR/node.tar.gz https://nodejs.org/dist/v4.2.4/node-v4.2.4-linux-x64.tar.gz
```

Comment:
We're assuming a 64-bit architecture here.  While I expect that's _probably_ correct, we should probably split out a separate installer or, preferably, detect the architecture near the top and modify the URL appropriately.